src/result_gen.py

```diff
@@ -561,10 +590,10 @@ def main():
         flag = measure_rg.check_enough_height_prediction()
         if not flag:
             continue
-        flag = measure_rg.create_result_in_json_format(model_id)
-        if not flag:
-            continue
+        measure_rg.create_result_in_json_format(model_id)
         measure_rg.update_measure_table_and_blob(model_id, destination_folder)
+        measure_rg.get_pose_results("posenet_1.0", "aci-posenet-ind")
```

Comment:
Make this also an an argument. similar to height model. Rename mode_id to height_model_id and pose_model_id

Reply:
added separate arguments for models